README.md

```diff
@@ -733,38 +749,41 @@ $ clarinet integrate
 
 Clarinet will spin-up a subnet node. More documentation on how to use and interact with this incoming L2 can be found on the [Hyperchain repository](https://github.com/hirosystems/stacks-subnets).
 
-## Contributing
+## Contributing to Clarinet
 
-We welcome contributions to Clarinet! The following sections provide information on how to contribute.
+Contribtions are welcome and appreciated. The following sections provide information on how you can contribute to Clarinet.
 
 ### Prerequisites
 
+Before contributing to Clarinet, please ensure you meet the following requirements:
+
 - rust (>=1.52.0)
 - cargo (>=1.52.0)
 - node (>=v14.16.0) - Used for git commit hook
 - npm (>=7.18.0) - Used for git commit hook
 
 ### Guide
 
-This repo follows the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) spec when writing commit messages.
-It's important any pull requests submitted have commit messages which follow this standard.
+This repo follows the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification when writing commit messages.
+
+**Note**It is important that any pull requests you submit have commit messages that follow this standard.
 
 To start contributing:
 
 1. Fork this repo and clone the fork locally.
-1. Create a new branch
+2. Create a new branch
    ```bash
    git checkout -b <my-branch>
    ```
-1. Run `npm i` in the local repo to install and initialize `husky` and `commitlint`.
+3. Run `npm i` in the local repo to install and initialize `husky` and `commitlint`.
```

Comment:
Same here `1.`,  and int the rest of this list

Reply:
fixed